lib/clients/github.go

```diff
@@ -167,7 +171,12 @@ func (g realGHClient) request(f func() (interface{}, *github.Response, error)) (
 	b := backoff.NewExponentialBackOff()
 	b.MaxElapsedTime = g.config.GetTimeout()
 
-	er := backoff.Retry(op, b)
+	er := backoff.RetryNotify(op, b, func(err error, duration time.Duration) {
+		duration /= 1000000 // Convert nanoseconds to milliseconds
+		duration *= 1000000 // Convert back so it appears correct
```

Comment:
Duration is an integer type; it truncates the lower 6 digits to zeros and creates a more readable number. Is there a better way you'd recommend?

I do agree though, but the minimum backoff is 500 milliseconds.